### PR TITLE
ENTESB-15380 Jira-source yaks test

### PR DESCRIPTION
**Pull request by lfabriko**

### CODE REVIEW

**Review by christophd — APPROVED** [review]
Looks good!

Just a small hint: The `oc apply` and the `kamel run` preparation steps could be moved to the feature itself. YAKS provides steps to apply any Kubernetes resources as `yaml`. You should also be able to run the Camel K integration as a step, too. We just need to add the property-file support here in YAKS.

File: test/jira/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Jira Kamelet test
+
+This test verifies the Jira Kamelet source defined in [jira-source.kamelet.yaml](jira-source.kamelet.yaml)
+
+## Objectives
+
+The test verifies the jira-source Kamelet by creating a Camel-K integration that uses the Kamelet and listens for new
```
Comment (apupier) [inline test/jira/README.md:7]:
```suggestion
The test verifies the jira-source Kamelet by creating a Camel K integration that uses the Kamelet and listens for new 
```

File: test/jira/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## Objectives
+
+The test verifies the jira-source Kamelet by creating a Camel-K integration that uses the Kamelet and listens for new 
+Jira issue objects. New issue object is passed to InMemoryChannel, which is bound to logger-sink Kamelet by KameletBinding.
+
+The test performs the following high level steps:
+
+*Preparation*
+- Create temporary namespace and install Yaks operator into it
+- Install Camel-K operator in temporary namespace
```
Comment (apupier) [inline test/jira/README.md:14]:
```suggestion
- Install Camel K operator in temporary namespace
```

File: test/jira/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# Jira Kamelet test
+
+This test verifies the Jira Kamelet source defined in [jira-source.kamelet.yaml](jira-source.kamelet.yaml)
```
Comment (apupier) [inline test/jira/README.md:3]: wrong link? do you mean logger-sink.kamelet.yaml?

File: test/jira/README.md (hunk 1 above)
Comment (lfabriko) [reply test/jira/README.md:7]: Surely I can change it. Version without hyphen complies e.g. with official documentation https://camel.apache.org/camel-k/latest/configuration/configuration.html.

File: test/jira/README.md (hunk 3 above)
Comment (lfabriko) [reply test/jira/README.md:3]: The link I provided is wrong - I really meant jira-source kamelet, so I should change it to https://github.com/openshift-integration/kamelet-catalog/blob/master/jira-source.kamelet.yaml.

**Pull request merged**